example/custom-metrics-deployment.yaml

```diff
@@ -72,13 +72,12 @@ spec:
       securityContext:
         seccompProfile:
           type: RuntimeDefault
-      serviceAccount: gardener-custom-metrics
       serviceAccountName: gardener-custom-metrics
       terminationGracePeriodSeconds: 30
       volumes:
         - name: gardener-custom-metrics-tls
           secret:
-            secretName: gardener-custom-metrics-tls
+            secretName: gardener-custom-metrics-tls-121d8b86 # TODO: Andrey: P1: This is dynamic
```

Comment:
This could still be static if the when the `garden-custom-metrics` is deployed from the examples, right?